app/graphql/mutations/eras/delete_era.rb

```diff
@@ -0,0 +1,14 @@
+module Mutations
+  module Eras
+    class DeleteEra < ::Mutations::BaseMutation
```

Comment:
```suggestion
class Mutations::Eras::DeleteEra < ::Mutations::BaseMutation
```
I think this follows the convention we used for the tests but I'm not sure if it would work! maybe give it a shot.

Reply:
I tried this syntax on UpdateEra and it works!